runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/emote/SingleItemRequirement.java

```diff
@@ -24,12 +24,14 @@
  */
 package net.runelite.client.plugins.cluescrolls.clues.emote;
 
+import lombok.Getter;
 import net.runelite.api.Client;
 import net.runelite.api.Item;
 import net.runelite.api.ItemComposition;
 
 public class SingleItemRequirement implements ItemRequirement
 {
+	@Getter
```

Comment:
if you use fullfiledBy you dont need to expose itemId

Reply:
Or well, maybe adding method fullfiledBy that just accepts int.. itemId

Reply:
yes